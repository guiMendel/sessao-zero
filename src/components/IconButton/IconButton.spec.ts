import { mountOptions } from '@/tests'
import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import { IconButton } from '.'

describe('IconButton', () => {
  it('should render properly', () => {
    const wrapper = mount(IconButton, mountOptions)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
