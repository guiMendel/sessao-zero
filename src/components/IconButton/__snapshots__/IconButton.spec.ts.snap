// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`IconButton > should render properly 1`] = `"<font-awesome-icon-stub data-v-520ca059=\\"\\" class=\\"button-icon\\" icon=\\"fas,\\"></font-awesome-icon-stub>"`;
